### ISSUE TITLE
HTTP Error 301: Moved Permanently

### ISSUE THREAD

**Issue by simvirakaiml** [jazzband/django-push-notifications#565]
I got this error while trying send notification to user.

**Comment by RealOrangeOne** [comment]
I assume you're trying to send a notification to an Android device? This happens because GCM shut down, and was replaced with firebase. You'll need to convert things to use Firebase rather than GCM.

**Comment by SerhiyRomanov** [comment]
@RealOrangeOne Does it mean that in the current version of this package the FCM sending doesn't work?

**Comment by RealOrangeOne** [comment]
@SerhiyRomanov No. It means sending with GCM is broken. Sending with FCM is just fine, although FCM isn't the default, so there's a couple more hoops to jump through.
